### PR TITLE
updated protocol

### PR DESCRIPTION
**Pull request by JeroenDMW**
Protocol updated to resemble official study protocol.

### CODE REVIEW

File: DynaMORE/protocol.json [hunk 1]
```diff
@@ -38,51 +38,75 @@
         "de": "",
         "es": ""
       },
-      "estimatedCompletionTime": 5,
+      "estimatedCompletionTime": 10,
       "protocol": {
         "repeatProtocol": {
           "unit": "day",
-          "amount": 4
+          "amount": 0
```
Comment (mpgxvii) [inline DynaMORE/protocol.json]: The repeatProtocol amount can't be zero. Did you want the protocol to not repeat at all or to repeat everyday?

Comment (JeroenDMW) [reply DynaMORE/protocol.json]: I do not want the protocol to repeat itself, as the entire 6-day schedule is defined in the 'repeatQuestionnaire' part

Comment (mpgxvii) [reply DynaMORE/protocol.json]: Currently, the app doesn't support protocols that don't repeat. There is a manually set end time (in the app) of 2 years for the protocol, so the protocol will repeat until the end time is reached. So, currently what can be done is setting the amount to a very high number. For example the `DEMO` questionnaire here: https://github.com/RADAR-base/RADAR-aRMT-protocols/blob/master/RADAR-MDD-KCL-s1/protocol.json#L205

Comment (JeroenDMW) [reply DynaMORE/protocol.json]: interesting, I will set the amount to 9999 years as done in the demo :)

**Comment by mpgxvii** [comment]
LGTM

**Pull request merged**